cypress/e2e/awx/administration/wfApprovalsList.cy.ts

```diff
@@ -0,0 +1,595 @@
+                          jobName = wfApprovalC.name.split(' ').slice(-1).toString();
+                          cy.filterTableBySingleSelect('name', jobName);
+                          cy.get('tbody').find('tr').should('have.length', 3);
+                          cy.getByDataCy('select-all').click();
+                          cy.getBy('[data-ouia-component-id="page-toolbar"]').within(() => {
+                            cy.getByDataCy(`${selectorDataCy}`).click();
+                          });
+                          cy.getModal().within(() => {
+                            cy.get('[data-ouia-component-id="confirm"]').click();
+                            cy.get('[data-ouia-component-id="submit"]').click();
+                            cy.clickButton('Close');
+                          });
+                          cy.waitForWorkflowJobStatus(launched.id.toString());
+                          cy.waitForWorkflowJobStatus(relaunched.id.toString());
+                          cy.waitForWorkflowJobStatus(relaunchedAgain.id.toString());
+                        }
+                      );
+                    });
+                }
+              );
+            });
+        });
+      });
+  }
+
+  /* 
+  Used in the Workflow Approvals - Bulk Approve, Bulk Deny, Bulk Delete tests (below)
+  **/
+  function deleteApprovalFromListToolbar() {
+    cy.get('tbody').find('tr').should('have.length', 3);
+    cy.getByDataCy('select-all').click();
+    cy.getBy('[data-ouia-component-id="page-toolbar"]').within(() => {
+      cy.getByDataCy('actions-dropdown')
+        .click()
+        .then(() => {
+          cy.getByDataCy('delete').click();
+        });
+    });
+    cy.getModal().within(() => {
+      cy.get('[data-ouia-component-id="confirm"]').click();
+      cy.get('[data-ouia-component-id="submit"]').click();
+      cy.clickButton('Close');
+    });
+    cy.get('h2').should('contain', 'No results found');
+  }
+
+  describe('Workflow Approvals - User Access', () => {
+    let workflowApproval: WorkflowApproval;
+
+    it('can assign normal user the access to approve a workflow approval from the list toolbar', () => {
+      cy.createAwxWorkflowJobTemplate({
+        name: 'E2E Workflow Approval-USER APPROVE-' + randomString(4),
+        organization: organization.id,
+        inventory: inventory.id,
+      }).then((wfjt) => {
+        workflowJobTemplate = wfjt;
+        cy.createAwxWorkflowVisualizerJobTemplateNode(workflowJobTemplate, jobTemplate).then(
+          (jtNode) => {
+            jobTemplateNode = jtNode;
+            cy.createAwxWorkflowVisualizerApprovalNode(workflowJobTemplate).then((appNode) => {
+              approvalWFNode = appNode;
+              cy.createWorkflowJTAlwaysNodeLink(approvalWFNode, jobTemplateNode);
+
+              cy.intercept(
+                'POST',
+                awxAPI`/workflow_job_templates/${workflowJobTemplate.id.toString()}/launch/`
+              ).as('launched');
+              cy.visit(
```

Comment:
We should not be using `cy.visit` and we should not be hard coding url paths in the the tests